arch/rv32i/src/lib.rs

```diff
@@ -15,29 +15,35 @@ pub mod support;
 pub mod syscall;
 extern crate tock_registers;
 
+#[cfg(target_arch = "riscv32")]
+pub const XLEN: usize = 32;
+// Default to 64-bit for CI
+#[cfg(any(target_arch = "riscv64", not(target_os = "none")))]
+pub const XLEN: usize = 64;
```

Comment:
Why default to 64 for CI?

Reply:
I suspect that most board will be 32-bit. The Kendryte board is an anomaly in that it has everything to support Linux but without an MMU.

Reply:
Ah it does! I know there was lots of debate if it had a non-spec MMU or just didn't have one.

Anyway, I think it's still not going to be a common occurrence having such a small board with a 64-bit CPU.